e2e/specs/st_columns.spec.js

```diff
@@ -23,28 +23,28 @@ describe("st.column", () => {
   it("creates 2 equal-width columns", () => {
     cy.get("[data-testid='stHorizontalBlock'] [data-testid='stBlock']")
       .eq(0)
-      .should("have.css", "flex", "1 1 calc(33.3333% - 16px)");
+      .should("have.css", "flex", "1 1 calc(33.3333% - 17px)");
```

Comment:
Curious on why this is 17 and not 18px?

Reply:
Because I'm an idiot. This should actually say `1rem`. Fixed!